src/components/productsList/_productsList.scss

```diff
@@ -0,0 +1,121 @@
+$media-tablet: 768px;
+$media-desktop: 1024px;
+$padding-mobile: 20px;
+$padding-desktop: 40px;
+
+.products-list {
+  display: flex;
+  flex-direction: column;
+  margin: 40px 10px 10px;
+
+  * {
+    box-sizing: border-box;
+  }
+
+  @media screen and (min-width: $media-desktop) {
+    flex-direction: row;
+  }
+}
+
+.products-list-nav {
+  $self: &;
+  background: $primary;
+  position: relative;
+  padding: $padding-mobile;
+  border-radius: 8px;
+
+  @media screen and (min-width: $media-tablet) {
+    padding: $padding-desktop;
+  }
+
+  @media screen and (min-width: $media-desktop) {
+    flex: 1 0 50%;
+    padding: calc(#{$padding-mobile} + 6%) $padding-desktop;
+  }
+  
+  &__title {
+    color: $secondary;
+    font-size: 16px;
+    font-weight: normal;
```

Comment:
Same goes for font-weights. Also, make sure that you know the difference between `normal` and an actual value like `400`